### PR TITLE
Tmz 1.6.0 calendar conversions

### PR DESCRIPTION
**Pull request by t-m-z**
converts between
- Julian Date
- Modified Julian Date
- Gregorian Calendar Date
- Julian Calendar Date

### CODE REVIEW

**Comment by t-m-z** [comment]
add tests to check logic
add calendar conversions for Hebrew, Coptic and Persian Calendars

File: assets/i18n/en.json [hunk 1]
```diff
@@ -1261,6 +1261,17 @@
   "dates_weekday_friday": "Friday",
   "dates_weekday_saturday": "Saturday",
   "dates_weekday_sunday": "Sunday",
+  "dates_calendar_title" : "Kalender",
+  "dates_calendar_description" : "Konvertieren von Datumsangaben",
```
Comment (S-Man42) [inline assets/i18n/en.json:1265]: English

File: lib/widgets/tools/science_and_technology/date_and_time/calendar.dart [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+      case CalendarSystem.PERSIANYAZDEGARDCALENDAR:
+      case CalendarSystem.COPTICCALENDAR:
+        if (verbose)
+          return date.day + ' ' + MONTH_NAMES[calendar][int.parse(date.month)].toString() + ' ' + date.year;
+        else
+          return date.day + '.' + date.month + '.' + date.year;
+        break;
+      case CalendarSystem.GREGORIANCALENDAR :
+      case CalendarSystem.JULIANCALENDAR :
+      switch (appLocale.languageCode) {
```
Comment (S-Man42) [inline lib/widgets/tools/science_and_technology/date_and_time/calendar.dart:165]: For these both types I would use the format lib of the system to fit the correct language format of the chosen display language. Do not construct the string by your own, please. This is already done here: https://github.com/S-Man42/GCWizard/blob/16ea8e9602619c5e0b0cac027411807ceed5f943/lib/widgets/main_menu/changelog.dart#L40

File: assets/i18n/en.json (hunk 1 above)
Comment (t-m-z) [reply assets/i18n/en.json:1265]: done

File: lib/widgets/tools/science_and_technology/date_and_time/calendar.dart (hunk 2 above)
Comment (t-m-z) [reply lib/widgets/tools/science_and_technology/date_and_time/calendar.dart:165]: not necessary because I always use the verbose mode

Comment (S-Man42) [reply lib/widgets/tools/science_and_technology/date_and_time/calendar.dart:165]: Hm, i got very szrange formats here

**Comment by t-m-z** [comment]
also refactor Maya to use date_utils
PR done

**Pull request merged**